frontend/src/app/admin/permit-application-list/permit-application-list.component.ts

```diff
@@ -16,13 +16,36 @@ export class PermitApplicationListComponent implements OnInit {
   applications: any;
   apiErrors: any;
 
-  constructor( private applicationService: ApplicationService ) { }
+  constructor( private applicationService: ApplicationService ) {
```

Comment:
Agreed, created issues [#545](https://github.com/flexion/fs-intake-module/issues/545) and [#546](https://github.com/flexion/fs-intake-module/issues/546).